schematic/store/synapse.py

```diff
@@ -77,35 +81,98 @@ class ManifestDownload(object):
     """
     syn: an object of type synapseclient.
     manifest_id: id of a manifest
+    synapse_entity_tracker: Tracker for a pull-through cache of Synapse entities
     """
 
     syn: synapseclient.Synapse
     manifest_id: str
+    synapse_entity_tracker: SynapseEntiyTracker = field(
+        default_factory=SynapseEntiyTracker
+    )
```

Comment:
Entity has a typo

Reply:
lol. I can't believe I hadn't noticed this before :D

Patched!